### ISSUE TITLE
Resolution

### ISSUE THREAD

**Issue by LukeIsCodingNet** [zorchenhimer/MovieNight#167]
Add a video resolution picker to the movie night video player.

**Comment by zorglube** [comment]
That is good idea. 
Underlying it need a multi-resolution stream, like HLS or something. 

Am I right ?

**Comment by CptPie** [comment]
> Am I right ?

Yes and no, the hls streaming is something zorch whipped up locally for his instance but it is not (yet) integrated into this repository.

Until it is the resolution picker is redundant since the codebase in this repository only supports one resolution (depending on the stream source).

**Comment by zorglube** [comment]
The other solution is make MovieNight encode stream on the fly.
